### PR TITLE
Update Sonarr for Config Flow

### PR DESCRIPTION
**Pull request by ctalkington**
## Proposed change

Update documentation to reflect config flow and config move to `sonarr` platform from `sensor` platform.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/33859
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1045 (already completed by someone else)
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/sonarr.markdown [hunk 1]
```diff
@@ -1,160 +1,19 @@
...
+ha_codeowners:
+  - '@ctalkington'
+ha_quality_scale: silver
 ---
 
-This `sonarr` sensor platform pulls data from a given [Sonarr](https://sonarr.tv/) instance.
+The `Sonarr` integration pulls data from a given [Sonarr](https://sonarr.tv/) instance.
 
-## Configuration
+### Configuration
```
Comment (klaasnicolaas) [inline source/_integrations/sonarr.markdown]:
```suggestion
## Configuration
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @ctalkington 👍

**Comment by agneevX** [comment]
Why was most of the documentation removed @ctalkington  ? Makes no sense.

**Comment by frenck** [comment]
@agneevX Because the removed documentation doesn't apply anymore. It will now be configured via the UI, YAML support has been removed from the integration and thus removed from the documentation.

**Comment by agneevX** [comment]
Right but there are no details or indication of what the integration has to offer. 

Might as well make it blank.

**Comment by frenck** [comment]
That wasn't there before either, only configuration has been removed. However, feel free to help out and open a PR to add it.

**Pull request merged**